wurst/systems/spells/Heat.wurst

```diff
@@ -13,7 +14,7 @@ function heat()
     end))
     for hero from heroes
         let owner = hero.getOwner()
-        owner.setGold(min(owner.getGold() + udg_HEAT_PER_CAST, udg_MAX_HEAT))
+        owner.setGold(min(owner.getGold() + gameConfig.getHeatPerCast(), gameConfig.getHeatMaximum()))
```

Comment:
I can't comment on a line outside of diff, but add a `TODO` on the `FIRE_RANGE` saying it needs to be synced to the ability range.

Reply:
@jonathanmclaus I think we should replace it with an aura similar to mud hut which gives you heat that exponentially increases every stat tick (1, 2, 4, 8, 16, 32, 64, 128). In any case I added the TODO